bin/utils/ngx_handle.lua

```diff
@@ -9,12 +9,12 @@ local function create_dirs(necessary_dirs)
 end
 
 local function remove_ngx_conf(ngx_conf)
-    os.remove(ngx_conf)
+    os.command("mv -f "..ngx_conf .. " /tmp/")
```

Comment:
why did  do this？

Reply:
in OPS rules , linux command rm is forbidden.

Reply:
can you give an example? os.remove is a Lua language package that is much safer than executing Linux commands directly.

Reply:
I see it , "os.rename(ngx_conf, "/tmp/"..ngx_conf)" is better then "os.remove(ngx_conf)"

Reply:
done

Reply:
OK